### ISSUE TITLE
Wrong device named in PGP key updated email

### ISSUE THREAD

**Issue by wjlafrance** [keybase/keybase-issues#3276]
I [updated my PGP key](https://keybase.io/wjlafrance/sigchain#2cdefdb18d34c15fd528e56ed319b0cdd87caca4d4060ae3c4880aa7c4ba70960f) from my Linux box ([`massachusetts-ubuntu`](https://keybase.io/wjlafrance/sigchain#4ff7aecc8556255e45fe38fe064201136b23c401f04bded0dca8bac911da3dee0f)). This shows on my sigchain as "performed no action" but from the payload it's clear that it was a PGP update.

I received an email confirming this, but which stated:

> You signed this change with **alaska-osx-2**.

This key was originally [uploaded to Keybase](https://keybase.io/wjlafrance/sigchain#7e30814e4e85a347676fd4fbeeb23f2a73c234d50c31f4b15e84fe91ec3dd2320f) last October using that machine, but that machine was not used to update the key.

I believe this is a bug in the email template, which either means to mention the device that originally created the key, or is pulling the wrong device name from the update.

**Comment by strib** [comment]
cc: @malgorithms

**Comment by malgorithms** [comment]
thanks for this issue @wjlafrance . Closing this out. Details:

- the site is now updated to say "updated PGP key" instead of "performed no action" on PGP key updates. This had been on our list for a while, so nice to get it closed.
- turns out there's a bug in the notification email and in certain cases can list the wrong device (as you saw). That fix is slightly more work and a separate internal ticket just started for it, so in the meantime, I've changed notifications not to mention the acting device when a PGP key is updated. This info is still available on the sigchain page, where it is correct. And the important thing is getting that email sent.

so yeah, gonna close this since we're no longer printing incorrect info. But we'll add back in the signing device info soon.
